fuzzers/klee/runner.Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/fuzzbench/base-runner
+FROM gcr.io/fuzzbench/base-image
```

Comment:
Does base-runner no longer exist?

Reply:
yes right, it does not since July, this is all old code that was working due to cache, i deleted it now.

Reply:
ACK. Thanks!